pre-commit and twine are not needed to run tests.

--- aiosignal-1.4.0/requirements/wheel.txt.orig
+++ aiosignal-1.4.0/requirements/wheel.txt
@@ -1,4 +1,2 @@
 pytest==8.4.1
 pytest-asyncio==1.0.0
-pre-commit==4.2.0
-twine==6.1.0
